libbeat/common/nomad/util.go

```diff
@@ -0,0 +1,82 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build linux darwin windows
+
+package nomad
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+
+	nomad "github.com/hashicorp/nomad/api"
+)
+
+var indexRegex = regexp.MustCompile(`\[(?P<index>[0-9]+)\]`)
+
+// FetchProperties returns a map with all the properties of the allocation
+func FetchProperties(alloc *nomad.Allocation) map[string]interface{} {
+	properties := map[string]interface{}{
+		"region":     *alloc.Job.Region,
+		"namespace":  alloc.Namespace,
+		"job":        alloc.JobID,
+		"group":      alloc.TaskGroup,
+		"allocation": alloc.ID,
+	}
+
+	if matchs := indexRegex.FindStringSubmatch(alloc.Name); len(matchs) == 2 {
```

Comment:
I assume it is a typo. You meant to write `matches`, right?

Reply:
Yep. Fixed